lib/Styles/Core.less

```diff
@@ -204,11 +204,7 @@ select {
 
   p {
     margin-top: 0;
-    margin-bottom: 1rem;
-  }
-
-  p:last-child {
-    margin-bottom: 0;
+    margin-bottom: 0; // 1rem;
```

Comment:
Normal styling has `margin-bottom: 1rem` not 0, right?  Plus, I think avoiding extra space after the last paragraph is a good thing anyway.